src/components/identity/global-header/GlobalHeader.jsx

```diff
@@ -21,6 +21,7 @@ export const GlobalHeaderProps = {
   mainNav: SBBlokType,
   hasHero: PropTypes.bool,
   isDark: PropTypes.bool,
+  searchBlockUrl: SBBlokType,
```

Comment:
```suggestion
  searchPageUrl: SBBlokType,
```
Looks like a typo here. The prop name is different from the one below.

Reply:
If replacing this field with Storyblok Link component, please change the prop type accordingly. There should be a custom prop type already created for Storyblok Link.